views/mdc/assets/js/components/tab-bars.js

```diff
@@ -10,5 +10,20 @@ export function initTabBars(e) {
 export class VTabBar extends eventHandlerMixin(VBaseComponent) {
     constructor(element, mdcComponent) {
         super(element, mdcComponent);
+
+        mdcComponent.listen('MDCTabBar:activated', (event) => {
+            console.log(event);
+            const tabs = element.parentElement.querySelectorAll('.v-tab-content');
+
+            tabs.forEach(function(element) {
```

Comment:
A for...of loop will be more performant and is (imo) more readable:
```suggestion
            for (const element of tabs) {
```